examples/lusid/instruments/upload-term-deposit-instruments.sql

```diff
@@ -2,7 +2,7 @@
 
 -- Description:
 -- 1. In this query, we run an ETL process on some instruments.
--- 2. First, we load a CSV file of instruments from Drive.
+-- 2. First, we load a Exel file of instruments from Drive.

```

Comment:
This is the tiniest little thing, but could you change this to Excel*, thanks

Reply:
There is always a typo to correct... fixed.